src/router/resolvers/index.ts

```diff
@@ -49,6 +50,16 @@ export const CurrentContractRouteResolver = (current: string): string => {
     return routeNames.Current_Contract;
   }
 
-  return routeNames.Performance_Requirements; 
+  return routeNames.Performance_Requirements;
 };
 
+
+export const PIIRecordResolver = (current: string): string => {
+  const systemOfRecord = PIIRecord.PIIRecordIncluded;
+  console.log(systemOfRecord)
+  // if system of record will be included, route to system of records page
+  if (current === routeNames.PII && systemOfRecord) {
+    return routeNames.PIIRecord;
+  }
+  return 'Public_Disclosure_of_Information';

```

Comment:
is there a `routeNames` equivalent for this?

Reply:
not yet

Reply:
just added it.